### DISCUSSION TITLE
Could not find header file "abb_librws_export.h"

### DISCUSSION THREAD

**Discussion by colinliew** [ros-industrial/abb_librws#169] [Q&A]
Hi, 

I am not able to find this file anywhere. Am i missing something?

#include "abb_librws_export.h"

Colin

**Comment by colinliew** [comment]
I'm trying to integrate the code in to my c++ code. I'm guessing the header is generated by catkin.

**Reply by gavanderhoorn** [reply]
`abb_librws` is a pure-CMake package.

Catkin is not used anywhere.

The header you refer to is generated by CMake, here:

https://github.com/ros-industrial/abb_librws/blob/9ba08371482f3975f364011db9b1368fa0fc6b32/CMakeLists.txt#L48-L49
